### ISSUE TITLE
[QoL Improvement] Print pending confirmations on Bitcoin lock for swap user

### ISSUE THREAD

**Issue by sethsimmons** [comit-network/xmr-btc-swap#671]
Currently when the swap user initially deposits funds their logs tell them they are waiting for Alice to lock Monero when they are in fact waiting on confirmations for the Bitcoin lock transaction (AFAIK).

If wrong about the process, please just close this out 🙂

**Comment by thomaseizinger** [comment]
> Currently when the swap user initially deposits funds their logs tell them they are waiting for Alice to lock Monero when they are in fact waiting on confirmations for the Bitcoin lock transaction (AFAIK).
> 
> If wrong about the process, please just close this out slightly_smiling_face

Well, semantically speaking, they are waiting for Alice to lock Monero. How many confirmations are required for Alice to do so depends on Alice.

I guess we could exchange this information at the beginning so we know what conf target Alice has.

**Comment by sethsimmons** [comment]
Yeah, IMO it's misleading to the swap client as it implies that they are only waiting on Alice to lock Monero, when Alice is waiting on their lock transaction to have the proper amount of confirmations.

Especially as there is a default confirmation value that most/all ASB owners seem to use, would be ideal if we can provide that info to the swap client so that the user knows what is taking so long on that step.
